mock-bpa-test/test_bpa.py

```diff
@@ -82,7 +76,7 @@ def setUp(self):
             '-s', 'ipn:2.1',  # security source
             '-u', 'localhost:4556', '-r', 'localhost:14556',
             '-o', 'localhost:24556', '-a', 'localhost:34556',
-            '-p', policy_config,
+            '-j' if is_json else "-p", policy_config,
```

Comment:
Can this be explained in the [API documentation](https://nasa-ammos.github.io/BSL/html/examples-and-mocks.html#mock-bpa) for clarity? It probably deserves a whole section to explain command options and one to explain policy management (as siblings to existing "Key Management").